index.yaml

```diff
@@ -1,9 +1,29 @@
 apiVersion: v1
 entries:
+  talend-api-microgateway:
+  - apiVersion: v1
+    appVersion: "1.4"
+    created: "2019-12-09T08:40:49.2912238+01:00"
+    description: A Helm chart for Talend API Microgateway to be run along with Talend
+      Remote Engine for Pipelines
+    digest: 93bc88cecd62c86f267c8bfff4cb92addbc4d5c5f9742e4237a24eecd82f2e91
+    home: https://www.talend.com/products/
+    icon: https://www.talend.com/wp-content/uploads/talend-logo.svg
+    keywords:
+    - Talend
+    - API
+    - microgateway
+    maintainers:
+    - name: Talend
+      url: https://www.talend.com/products/
+    name: talend-api-microgateway
+    urls:
+    - https://talend.github.io/helm-charts-public/talend-api-microgateway-0.1.1.tgz
+    version: 0.1.1
   vault-sidecar-injector:
```

Comment:
@elogeart Do we really use the `vault-sidecar-injector`?
To my understanding: no; Thus we can remove that reference, WDYT?

Reply:
@pierCo No you can't. If you do so then Vault Sidecar Injector chart will not be refrenced anymore then not visible on Helm Hub.

Reply:
No we can't remove it, this quite new repo is the whole Talend Helm Charts public registry, not only the one for Talend API Microgateway 😉

Reply:
No do not remove it please, this is an open source project that we maintain.